### PR TITLE
[ios] Integrate Mapbox Mobile Events 0.9.4

### LINKED ISSUE #14867: [ios] integrate Mapbox Mobile Events 0.9.4
Target is: iOS alpha2
Mapbox Mobile Events 0.9.4: https://github.com/mapbox/mobile-telemetry/issues/320

### PR DESCRIPTION
**Pull request by alfwatt**
Integrate: https://github.com/mapbox/mapbox-events-ios/releases/tag/v0.9.4

Fixes #14867.

### CODE REVIEW

File: platform/ios/ios.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1805,6 +1856,65 @@
...
+				9C6E285422A984710056B7BE /* package.sh */,
+				9C6E285522A984710056B7BE /* trigger-external-deploy-steps.sh */,
+				9C6E285622A984710056B7BE /* release-notes-jazzy.md.ejs */,
+				9C6E285722A984710056B7BE /* deploy-snapshot.sh */,
+				9C6E285822A984710056B7BE /* release-notes.js */,
+			);
+			path = scripts;
+			sourceTree = "<group>";
+		};
+		9C6E285922A9849E0056B7BE /* scripts */ = {
```
Comment (friedbunny) [inline platform/ios/ios.xcodeproj/project.pbxproj]: This evidently adds the “scripts” folder twice:

Screen Shot 2019-06-06 at 10 54 42 AM

Comment (friedbunny) [reply platform/ios/ios.xcodeproj/project.pbxproj]: Also, if you could please capitalize Scripts, that’d soothe my frazzled pedant nerves. 🙇

**Comment by friedbunny** [comment]
Ticketed the `node-gcc8-debug` failure in https://github.com/mapbox/mapbox-gl-native/issues/14869 and restarted the job.

File: platform/ios/ios.xcodeproj/project.pbxproj (hunk 1 above)
Comment (alfwatt) [reply platform/ios/ios.xcodeproj/project.pbxproj]: It's lower-case on the filesystem

Comment (alfwatt) [reply platform/ios/ios.xcodeproj/project.pbxproj]: But we can be case-insensitive in our usage

**Pull request merged**